lib/telephone_number/test_data_generator.rb

```diff
@@ -0,0 +1,91 @@
+module TelephoneNumber
+  require 'nokogiri'
+  require 'httparty'
+  require 'yaml'
+
+  class TestDataGenerator
```

Comment:
Umm.. idk. It was intended to be a quick and easy way for pulling down the appropriate data for new tests cases. It was also intended to quickly and easily pull down the latest data.